Auto update failing for rbw

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [rbw](https://github.com/termux/termux-packages/blob/1841f30e61508eac55d5b532a2d70b389dcaf7ae/packages/rbw) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating rbw [Current version: 1.12.1]
INFO: package being updated to 1.13.0.
Downloading https://github.com/doy/rbw/archive/refs/tags/1.13.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  110k    0  110k    0     0   362k      0 --:--:-- --:--:-- --:--:--  362k

You are about to commit these changes:

--------------------
diff --git a/packages/rbw/build.sh b/packages/rbw/build.sh
index aedf51871c..563c5efbb0 100644
--- a/packages/rbw/build.sh
+++ b/packages/rbw/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/doy/rbw
 TERMUX_PKG_DESCRIPTION="An unofficial command line client for Bitwarden"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.12.1"
+TERMUX_PKG_VERSION="1.13.0"
 TERMUX_PKG_SRCURL=https://github.com/doy/rbw/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=c564484f1054a85014b6b2a1fbade24d56b1b221dbac681c682ffaeba158b697
+TERMUX_PKG_SHA256=7fa0e4a29d0c5d9225f5da4d461498ed9b1bef2bd0d194c2fdc7a57d41998e06
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="pinentry"
 TERMUX_PKG_BUILD_IN_SRC=true
--------------------

bump(main/rbw): 1.13.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package rbw exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building rbw for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13134  100 13134    0     0  31613      0 --:--:-- --:--:-- --:--:-- 31648
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2019      0 --:--:-- --:--:-- --:--:--  2016
gpg: Signature made Fri 27 Dec 2024 12:52:18 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  32434      0 --:--:-- --:--:-- --:--:-- 32487
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   1996      0 --:--:-- --:--:-- --:--:--  1992
100   833  100   833    0     0   1995      0 --:--:-- --:--:-- --:--:--  1992
gpg: Signature made Tue 17 Dec 2024 06:32:46 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13120  100 13120    0     0  23288      0 --:--:-- --:--:-- --:--:-- 23303
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2013      0 --:--:-- --:--:-- --:--:--  2012
100   833  100   833    0     0   2012      0 --:--:-- --:--:-- --:--:--  2012
gpg: Signature made Fri 27 Dec 2024 06:10:26 AM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/main/binary-aarch64/Packages
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
 17  500k   17 88711    0     0   111k      0  0:00:04 --:--:--  0:00:04  111k
100  500k  100  500k    0     0   486k      0  0:00:01  0:00:01 --:--:--  486k
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libandroid-support/libandroid-support_29_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 10852  100 10852    0     0   252k      0 --:--:-- --:--:-- --:--:--  258k
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libgpg-error@1.50 if necessary...
Found libgpg-error in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libg/libgpg-error/libgpg-error_1.50_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  111k  100  111k    0     0  2554k      0 --:--:-- --:--:-- --:--:-- 2597k
extracting libgpg-error to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libassuan@3.0.1-2 if necessary...
Found libassuan in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/liba/libassuan/libassuan_3.0.1-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 73896  100 73896    0     0  1568k      0 --:--:-- --:--:-- --:--:-- 1603k
extracting libassuan to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libiconv@1.17 if necessary...
Found libiconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/libi/libiconv/libiconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  543k  100  543k    0     0  1266k      0 --:--:-- --:--:-- --:--:-- 1269k
extracting libiconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency iconv@1.17 if necessary...
Found iconv in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/i/iconv/iconv_1.17_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  8988  100  8988    0     0   210k      0 --:--:-- --:--:-- --:--:--  214k
extracting iconv to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses/ncurses_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  525k  100  525k    0     0  10.2M      0 --:--:-- --:--:-- --:--:-- 10.2M
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-utils/ncurses-utils_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 87728  100 87728    0     0  1982k      0 --:--:-- --:--:-- --:--:-- 1992k
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs/ncurses-ui-libs_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 32964  100 32964    0     0   663k      0 --:--:-- --:--:-- --:--:--  670k
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/n/ncurses-ui-libs-static/ncurses-ui-libs-static_6.5.20240831-2_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 39644  100 39644    0     0   953k      0 --:--:-- --:--:-- --:--:--  967k
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pinentry@1.3.1-1 if necessary...
Found pinentry in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pinentry/pinentry_1.3.1-1_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 46904  100 46904    0     0  1084k      0 --:--:-- --:--:-- --:--:-- 1090k
extracting pinentry to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/doy/rbw/archive/refs/tags/1.13.0.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  110k    0  110k    0     0   530k      0 --:--:-- --:--:-- --:--:--  530k
info: downloading installer
info: profile set to 'default'
info: default host triple is x86_64-unknown-linux-gnu
warning: Updating existing toolchain, profile choice will be ignored
info: syncing channel updates for '1.83.0-x86_64-unknown-linux-gnu'
info: default toolchain set to '1.83.0-x86_64-unknown-linux-gnu'

  1.83.0-x86_64-unknown-linux-gnu unchanged - rustc 1.83.0 (90b35a623 2024-11-26)


Rust is installed now. Great!

To get started you may need to restart your current shell.
This would reload your PATH environment variable to include
Cargo's bin directory ($HOME/.cargo/bin).

To configure your current shell, you need to source
the corresponding env file under $HOME/.cargo.

This is usually done by running one of the following (note the leading DOT):
. "$HOME/.cargo/env"            # For sh/bash/zsh/ash/dash/pdksh
source "$HOME/.cargo/env.fish"  # For fish
info: component 'rust-std' for target 'aarch64-linux-android' is up to date
    Updating crates.io index
 Downloading crates ...
  Downloaded anstyle-query v1.1.2
  Downloaded anstyle v1.0.10
  Downloaded derive-new v0.5.9
  Downloaded is-wsl v0.4.0
  Downloaded is-docker v0.2.0
  Downloaded want v0.3.1
  Downloaded write16 v1.0.0
  Downloaded wl-clipboard-rs v0.7.0
  Downloaded urlencoding v2.1.3
  Downloaded utf8_iter v1.0.4
  Downloaded wayland-scanner v0.29.5
  Downloaded uuid v1.11.0
  Downloaded libc v0.2.169
  Downloaded wayland-sys v0.29.5
  Downloaded url v2.5.4
  Downloaded untrusted v0.9.0
  Downloaded wayland-commons v0.29.5
  Downloaded unicode-ident v1.0.14
  Downloaded tungstenite v0.24.0
  Downloaded tracing v0.1.41
  Downloaded tower v0.5.2
  Downloaded regex v1.11.1
  Downloaded rsa v0.9.7
  Downloaded async-trait v0.1.83
  Downloaded serde_json v1.0.134
  Downloaded reqwest v0.12.9
  Downloaded rustls v0.23.20
  Downloaded quinn-proto v0.11.9
  Downloaded tracing-core v0.1.33
  Downloaded syn v2.0.91
  Downloaded rustls-pki-types v1.10.1
  Downloaded quinn-udp v0.5.9
  Downloaded icu_locid_transform_data v1.5.0
  Downloaded cpufeatures v0.2.16
  Downloaded base64 v0.22.1
  Downloaded typenum v1.17.0
  Downloaded rustix v0.38.42
  Downloaded tinyvec v1.8.1
  Downloaded socket2 v0.5.8
  Downloaded sha1 v0.10.6
  Downloaded serde_derive v1.0.216
  Downloaded rustls-pemfile v2.2.0
  Downloaded rmp v0.8.14
  Downloaded pkcs8 v0.10.2
  Downloaded yoke v0.7.5
  Downloaded totp-lite v2.0.1
  Downloaded tokio-tungstenite v0.24.0
  Downloaded tokio-stream v0.1.17
  Downloaded tokio-rustls v0.26.1
  Downloaded tempfile v3.14.0
  Downloaded sha2 v0.10.8
  Downloaded petgraph v0.6.5
  Downloaded pkcs1 v0.7.5
  Downloaded pem-rfc7468 v0.7.0
  Downloaded pbkdf2 v0.12.2
  Downloaded paste v1.0.15
  Downloaded tokio v1.42.0
  Downloaded os_pipe v1.2.1
  Downloaded num-traits v0.2.19
  Downloaded num-iter v0.1.45
  Downloaded httparse v1.9.5
  Downloaded futures-sink v0.3.31
  Downloaded futures-channel v0.3.31
  Downloaded try-lock v0.2.5
  Downloaded tree_magic_mini v3.1.6
  Downloaded tower-service v0.3.3
  Downloaded tower-layer v0.3.3
  Downloaded thiserror-impl v2.0.9
  Downloaded thiserror-impl v1.0.69
  Downloaded thiserror v1.0.69
  Downloaded smawk v0.3.2
  Downloaded serde_path_to_error v0.1.16
  Downloaded serde v1.0.216
  Downloaded rustls-webpki v0.102.8
  Downloaded proc-macro2 v1.0.92
  Downloaded pathdiff v0.2.3
  Downloaded parking_lot v0.12.3
  Downloaded openssl-probe v0.1.5
  Downloaded hmac v0.12.1
  Downloaded tokio-macros v2.4.0
  Downloaded tinystr v0.7.6
  Downloaded thiserror v2.0.9
  Downloaded textwrap v0.16.1
  Downloaded terminal_size v0.4.1
  Downloaded synstructure v0.13.1
  Downloaded sync_wrapper v1.0.2
  Downloaded subtle v2.6.1
  Downloaded spki v0.7.3
  Downloaded spin v0.9.8
  Downloaded smallvec v1.13.2
  Downloaded slab v0.4.9
  Downloaded signal-hook-registry v1.4.2
  Downloaded serde_repr v0.1.19
  Downloaded scopeguard v1.2.0
  Downloaded rustc-hash v2.1.0
  Downloaded region v3.0.2
  Downloaded regex-automata v0.4.9
  Downloaded quinn v0.11.6
  Downloaded pkg-config v0.3.31
  Downloaded pin-utils v0.1.0
  Downloaded pin-project-lite v0.2.15
  Downloaded password-hash v0.5.0
  Downloaded parking_lot_core v0.9.10
  Downloaded open v5.3.1
  Downloaded zerovec-derive v0.10.3
  Downloaded zerovec v0.10.4
  Downloaded zerofrom-derive v0.1.5
  Downloaded zerofrom v0.1.5
  Downloaded yoke-derive v0.7.5
  Downloaded syn v1.0.109
  Downloaded stable_deref_trait v1.2.0
  Downloaded signature v2.2.0
  Downloaded serde_urlencoded v0.7.1
  Downloaded rustversion v1.0.18
  Downloaded rustls-native-certs v0.8.1
  Downloaded rmpv v1.3.0
  Downloaded quote v1.0.38
  Downloaded nix v0.24.3
  Downloaded utf16_iter v1.0.5
  Downloaded utf-8 v0.7.6
  Downloaded wayland-client v0.29.5
  Downloaded wayland-protocols v0.29.5
  Downloaded icu_properties_data v1.5.0
  Downloaded hyper v1.5.2
  Downloaded hkdf v0.12.4
  Downloaded hashbrown v0.15.2
  Downloaded futures-util v0.3.31
  Downloaded clap_builder v4.5.23
  Downloaded axum v0.7.9
  Downloaded x11rb-protocol v0.10.0
  Downloaded x11rb v0.10.1
  Downloaded num-bigint-dig v0.8.4
  Downloaded nom v7.1.3
  Downloaded mio v1.0.3
  Downloaded minimal-lexical v0.2.1
  Downloaded libm v0.2.11
  Downloaded idna v1.0.3
  Downloaded icu_properties v1.5.1
  Downloaded http v1.2.0
  Downloaded aes v0.8.4
  Downloaded x11-clipboard v0.7.1
  Downloaded indexmap v2.7.0
  Downloaded icu_collections v1.5.0
  Downloaded displaydoc v0.2.5
  Downloaded der v0.7.9
  Downloaded cc v1.2.5
  Downloaded bytes v1.9.0
  Downloaded blake2 v0.10.6
  Downloaded is-terminal v0.4.13
  Downloaded ipnet v2.10.1
  Downloaded icu_provider v1.5.0
  Downloaded icu_normalizer_data v1.5.0
  Downloaded icu_normalizer v1.5.0
  Downloaded icu_locid v1.5.0
  Downloaded hyper-util v0.1.10
  Downloaded futures v0.3.31
  Downloaded env_logger v0.11.6
  Downloaded data-encoding v2.6.0
  Downloaded const-oid v0.9.6
  Downloaded clap_complete v4.5.40
  Downloaded clap v4.5.23
  Downloaded base64ct v1.6.0
  Downloaded axum-core v0.4.5
  Downloaded argon2 v0.5.3
  Downloaded anyhow v1.0.95
  Downloaded unicode-linebreak v0.1.5
  Downloaded xml-rs v0.8.24
  Downloaded writeable v0.5.5
  Downloaded num-integer v0.1.46
  Downloaded mime v0.3.17
  Downloaded memoffset v0.6.5
  Downloaded matchit v0.7.3
  Downloaded lock_api v0.4.12
  Downloaded litemap v0.7.4
  Downloaded itoa v1.0.14
  Downloaded inout v0.1.3
  Downloaded idna_adapter v1.2.0
  Downloaded icu_locid_transform v1.5.0
  Downloaded hyper-rustls v0.27.5
  Downloaded humantime v2.1.0
  Downloaded gethostname v0.2.3
  Downloaded generic-array v0.14.7
  Downloaded futures-task v0.3.31
  Downloaded futures-executor v0.3.31
  Downloaded futures-core v0.3.31
  Downloaded fixedbitset v0.4.2
  Downloaded fastrand v2.3.0
  Downloaded crypto-common v0.1.6
  Downloaded colorchoice v1.0.3
  Downloaded cipher v0.4.4
  Downloaded cbc v0.1.2
  Downloaded block-padding v0.3.3
  Downloaded base32 v0.5.1
  Downloaded autocfg v1.4.0
  Downloaded anstream v0.6.18
  Downloaded http-body-util v0.1.2
  Downloaded http-body v1.0.1
  Downloaded futures-macro v0.3.31
  Downloaded futures-io v0.3.31
  Downloaded fnv v1.0.7
  Downloaded errno v0.3.10
  Downloaded env_filter v0.1.3
  Downloaded downcast-rs v1.2.1
  Downloaded dirs-sys v0.4.0
  Downloaded directories v5.0.0
  Downloaded digest v0.10.7
  Downloaded cli-clipboard v0.4.0
  Downloaded clap_lex v0.7.4
  Downloaded block-buffer v0.10.4
  Downloaded bitflags v1.3.2
  Downloaded arrayvec v0.7.6
  Downloaded icu_provider_macros v1.5.0
  Downloaded daemonize v0.5.0
  Downloaded anstyle-parse v0.2.6
  Downloaded ring v0.17.8
   Compiling proc-macro2 v1.0.92
   Compiling unicode-ident v1.0.14
   Compiling libc v0.2.169
   Compiling autocfg v1.4.0
   Compiling cfg-if v1.0.0
   Compiling typenum v1.17.0
   Compiling version_check v0.9.5
   Compiling quote v1.0.38
   Compiling generic-array v0.14.7
   Compiling syn v2.0.91
   Compiling once_cell v1.20.2
   Compiling itoa v1.0.14
   Compiling smallvec v1.13.2
   Compiling subtle v2.6.1
   Compiling memchr v2.7.4
   Compiling getrandom v0.2.15
   Compiling const-oid v0.9.6
   Compiling pin-project-lite v0.2.15
   Compiling crypto-common v0.1.6
   Compiling stable_deref_trait v1.2.0
   Compiling bytes v1.9.0
   Compiling futures-core v0.3.31
   Compiling slab v0.4.9
   Compiling block-buffer v0.10.4
   Compiling lock_api v0.4.12
   Compiling futures-sink v0.3.31
   Compiling parking_lot_core v0.9.10
   Compiling futures-channel v0.3.31
   Compiling digest v0.10.7
   Compiling zeroize v1.8.1
   Compiling pin-utils v0.1.0
   Compiling scopeguard v1.2.0
   Compiling shlex v1.3.0
   Compiling futures-io v0.3.31
   Compiling futures-task v0.3.31
   Compiling cc v1.2.5
   Compiling fnv v1.0.7
   Compiling writeable v0.5.5
   Compiling litemap v0.7.4
   Compiling log v0.4.22
   Compiling spin v0.9.8
   Compiling http v1.2.0
   Compiling parking_lot v0.12.3
   Compiling ring v0.17.8
   Compiling synstructure v0.13.1
   Compiling socket2 v0.5.8
   Compiling signal-hook-registry v1.4.2
   Compiling mio v1.0.3
   Compiling byteorder v1.5.0
   Compiling rand_core v0.6.4
   Compiling icu_locid_transform_data v1.5.0
   Compiling httparse v1.9.5
   Compiling zerofrom-derive v0.1.5
   Compiling yoke-derive v0.7.5
   Compiling zerovec-derive v0.10.3
   Compiling displaydoc v0.2.5
   Compiling futures-macro v0.3.31
   Compiling zerofrom v0.1.5
   Compiling yoke v0.7.5
   Compiling tokio-macros v2.4.0
   Compiling zerovec v0.10.4
   Compiling futures-util v0.3.31
   Compiling icu_provider_macros v1.5.0
   Compiling tokio v1.42.0
   Compiling libm v0.2.11
   Compiling rustls-pki-types v1.10.1
   Compiling tinystr v0.7.6
   Compiling icu_locid v1.5.0
   Compiling icu_collections v1.5.0
   Compiling icu_provider v1.5.0
   Compiling zerocopy-derive v0.7.35
   Compiling icu_locid_transform v1.5.0
   Compiling num-traits v0.2.19
   Compiling base64ct v1.6.0
   Compiling icu_properties_data v1.5.0
   Compiling untrusted v0.9.0
   Compiling icu_properties v1.5.1
   Compiling zerocopy v0.7.35
   Compiling http-body v1.0.1
   Compiling tracing-core v0.1.33
   Compiling utf16_iter v1.0.5
   Compiling icu_normalizer_data v1.5.0
   Compiling rustix v0.38.42
   Compiling write16 v1.0.0
   Compiling serde v1.0.216
   Compiling utf8_iter v1.0.4
   Compiling percent-encoding v2.3.1
   Compiling rustls v0.23.20
   Compiling form_urlencoded v1.2.1
   Compiling icu_normalizer v1.5.0
   Compiling tracing v0.1.41
   Compiling ppv-lite86 v0.2.20
   Compiling rustls-webpki v0.102.8
   Compiling serde_derive v1.0.216
   Compiling errno v0.3.10
   Compiling cpufeatures v0.2.16
   Compiling bitflags v2.6.0
   Compiling tower-service v0.3.3
   Compiling utf8parse v0.2.2
   Compiling try-lock v0.2.5
   Compiling linux-raw-sys v0.4.14
   Compiling want v0.3.1
   Compiling anstyle-parse v0.2.6
   Compiling rand_chacha v0.3.1
   Compiling idna_adapter v1.2.0
   Compiling pem-rfc7468 v0.7.0
   Compiling anstyle-query v1.1.2
   Compiling anstyle v1.0.10
   Compiling httpdate v1.0.3
   Compiling is_terminal_polyfill v1.70.1
   Compiling colorchoice v1.0.3
   Compiling anstream v0.6.18
   Compiling hyper v1.5.2
   Compiling der v0.7.9
   Compiling idna v1.0.3
   Compiling rand v0.8.5
   Compiling sync_wrapper v1.0.2
   Compiling block-padding v0.3.3
   Compiling aho-corasick v1.1.3
   Compiling serde_json v1.0.134
   Compiling paste v1.0.15
   Compiling rustversion v1.0.18
   Compiling thiserror v1.0.69
   Compiling openssl-probe v0.1.5
   Compiling regex-syntax v0.8.5
   Compiling ryu v1.0.18
   Compiling rustls-native-certs v0.8.1
   Compiling inout v0.1.3
   Compiling spki v0.7.3
   Compiling url v2.5.4
   Compiling regex-automata v0.4.9
   Compiling hyper-util v0.1.10
   Compiling tokio-rustls v0.26.1
   Compiling terminal_size v0.4.1
   Compiling sha1 v0.10.6
   Compiling num-integer v0.1.46
   Compiling http-body-util v0.1.2
   Compiling thiserror-impl v1.0.69
   Compiling hmac v0.12.1
   Compiling strsim v0.11.1
   Compiling mime v0.3.17
   Compiling num-bigint-dig v0.8.4
   Compiling clap_lex v0.7.4
   Compiling tower-layer v0.3.3
   Compiling heck v0.5.0
   Compiling clap_builder v4.5.23
   Compiling clap_derive v4.5.18
   Compiling num-iter v0.1.45
   Compiling regex v1.11.1
   Compiling pkcs8 v0.10.2
   Compiling cipher v0.4.4
   Compiling serde_urlencoded v0.7.1
   Compiling async-trait v0.1.83
   Compiling lazy_static v1.5.0
   Compiling is-docker v0.2.0
   Compiling anyhow v1.0.95
   Compiling utf-8 v0.7.6
   Compiling data-encoding v2.6.0
   Compiling tungstenite v0.24.0
   Compiling is-wsl v0.4.0
   Compiling axum-core v0.4.5
   Compiling rmp v0.8.14
   Compiling pkcs1 v0.7.5
   Compiling env_filter v0.1.3
   Compiling clap v4.5.23
   Compiling tower v0.5.2
   Compiling hyper-rustls v0.27.5
   Compiling serde_path_to_error v0.1.16
   Compiling sha2 v0.10.8
   Compiling password-hash v0.5.0
   Compiling futures-executor v0.3.31
   Compiling rustls-pemfile v2.2.0
   Compiling signature v2.2.0
   Compiling blake2 v0.10.6
   Compiling dirs-sys v0.4.0
   Compiling ipnet v2.10.1
   Compiling smawk v0.3.2
   Compiling pathdiff v0.2.3
   Compiling base64 v0.22.1
   Compiling fastrand v2.3.0
   Compiling matchit v0.7.3
   Compiling unicode-width v0.1.14
   Compiling humantime v2.1.0
   Compiling unicode-linebreak v0.1.5
   Compiling bitflags v1.3.2
   Compiling region v3.0.2
   Compiling textwrap v0.16.1
   Compiling env_logger v0.11.6
   Compiling axum v0.7.9
   Compiling reqwest v0.12.9
   Compiling tempfile v3.14.0
   Compiling open v5.3.1
   Compiling argon2 v0.5.3
   Compiling directories v5.0.0
   Compiling rsa v0.9.7
   Compiling futures v0.3.31
   Compiling totp-lite v2.0.1
   Compiling clap_complete v4.5.40
   Compiling rmpv v1.3.0
   Compiling tokio-tungstenite v0.24.0
   Compiling cbc v0.1.2
   Compiling aes v0.8.4
   Compiling pbkdf2 v0.12.2
   Compiling hkdf v0.12.4
   Compiling tokio-stream v0.1.17
   Compiling serde_repr v0.1.19
   Compiling uuid v1.11.0
   Compiling daemonize v0.5.0
   Compiling is-terminal v0.4.13
   Compiling arrayvec v0.7.6
   Compiling cli-clipboard v0.4.0
error[E0433]: failed to resolve: use of undeclared type `ClipboardContext`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cli-clipboard-0.4.0/src/lib.rs:123:19
    |
123 |     let mut ctx = ClipboardContext::new()?;
    |                   ^^^^^^^^^^^^^^^^ use of undeclared type `ClipboardContext`

error[E0433]: failed to resolve: use of undeclared type `ClipboardContext`
   --> /home/builder/.cargo/registry/src/index.crates.io-6f17d22bba15001f/cli-clipboard-0.4.0/src/lib.rs:142:19
    |
142 |     let mut ctx = ClipboardContext::new()?;
    |                   ^^^^^^^^^^^^^^^^ use of undeclared type `ClipboardContext`

For more information about this error, try `rustc --explain E0433`.
error: could not compile `cli-clipboard` (lib) due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-12-27 06:13:33 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/12512537553">12512537553</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/rbw/build.sh
+++ b/packages/rbw/build.sh
@@ -2,9 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://github.com/doy/rbw
 TERMUX_PKG_DESCRIPTION="An unofficial command line client for Bitwarden"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.12.1"
+TERMUX_PKG_VERSION="1.13.1"
 TERMUX_PKG_SRCURL=https://github.com/doy/rbw/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
-TERMUX_PKG_SHA256=c564484f1054a85014b6b2a1fbade24d56b1b221dbac681c682ffaeba158b697
+TERMUX_PKG_SHA256=2e3181417732b5ab32456a1babff2febeee695604e85db2c94668270ed8a2036
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_DEPENDS="pinentry"
 TERMUX_PKG_BUILD_IN_SRC=true
@@ -12,7 +12,11 @@ TERMUX_PKG_BUILD_IN_SRC=true
 termux_step_make() {
 	termux_setup_rust
 
-	cargo build --jobs $TERMUX_PKG_MAKE_PROCESSES --target $CARGO_TARGET_NAME --release
+	cargo build \
+		--jobs "$TERMUX_PKG_MAKE_PROCESSES" \
+		--target "$CARGO_TARGET_NAME" \
+		--no-default-features \
+		--release
 }
 
 termux_step_make_install() {

